Documentation/ApiOverview/LinkBrowser/Tutorials/_CustomLinkBrowser/_GithubLinkBuilder.rst.txt

```diff
@@ -0,0 +1,42 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
+.. Extracted from T3docs\Examples\LinkHandler\GithubLinkBuilder
+
+.. code-block:: php
+   :caption: Class T3docs\\Examples\\LinkHandler\\GithubLinkBuilder
+
+   use TYPO3\CMS\Frontend\Typolink\LinkResult;
+   use TYPO3\CMS\Frontend\Typolink\LinkResultInterface;
+   use TYPO3\CMS\Frontend\Typolink\UnableToLinkException;
+   
+   /**
+    * Builds a TypoLink to a Github issue
+    */
+   class GithubLinkBuilder extends AbstractTypolinkBuilder
+   {
+       private const TYPE_GITHUB = 'github';
+   
+       public function build(
+           array &$linkDetails,
```

Comment:
Why linkDetails by reference?

Reply:
The method signature is defined like that in AbstractTypolinkBuilder

Reply:
Puuh, old and bad code...